src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -82,7 +83,11 @@
  * </li>
  * </ol>
  * <p>
- * Use the separator '###' between rules.
+ * Rules are configured as a comma-separated ordered list.
+ * </p>
+ * <p>
+ * Note: '###' rule separator is deprecated but currently supported for backward
```

Comment:
Can we be a bit more specific.
`'###' rule separator is deprecated` -> `'###' rule separator, in place of the current commas, is deprecated`

Reply:
Done, changed the wording a bit